src/Uno.UI.Runtime.Skia.MacOS/MacOSWindowHost.cs

```diff
@@ -449,6 +449,7 @@ private static PointerEventArgs BuildPointerArgs(NativeMouseEventData data)
 		var pointerDevice = PointerDevice.For(data.PointerDeviceType);
 		var properties = GetPointerProperties(data).SetUpdateKindFromPrevious(_previousProperties);
 
+		// TODO: Check the macOS unit of time
```

Comment:
It's now [1] an `NSTimeInterval` (`double` measured in seconds) [2] multiplied by `10`. For microseconds it should be changed to `1000`.

https://github.com/unoplatform/uno/blob/4dd818472c94c70dc5d4c6237480130bd41e8e6b/src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOWindow.m#L912-L914

[1] I had to change that recently (removing a multiplication by `1000`) since it was overflowing and made the drag-n-drop only works the first time.

[2] https://developer.apple.com/documentation/foundation/nstimeinterval

Reply:
I have adjusted the code, seems ok and manual test passes!